tests/unit/app/test_writers.py

```diff
@@ -71,11 +79,22 @@ def test_s3_writer(df_lending_club, moto_boto, s3_all_config_path):
 
     client = boto3.client("s3")
     objects = client.list_objects(Bucket="mocked_bucket")
-
     for idx, each_objc in enumerate(objects["Contents"]):
         assert each_objc["Key"] == object_keys[idx]
 
 
+def test_s3_writer_transport_config(df_lending_club, s3_transport_config_path):
```

Comment:
it doesn't look like you are using the df_lending_club here, can you remove it?

Reply:
```suggestion
def test_s3_writer_transport_config(s3_transport_config_path):
```